sources/packages/backend/apps/api/src/app.external.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from "@nestjs/common";
+import { StudentExternalController } from "./route-controllers";
+import { AuthModule } from "./auth/auth.module";
+import { StudentService } from "@sims/integrations/services";
+
+@Module({
+  imports: [AuthModule],
+  controllers: [StudentExternalController],
+  providers: [StudentService],
```

Comment:
Open for discussion but we can keep those services at the API scope at first.
We can create some specific services for the external module, but I am not sure if we should start creating them in the integrations lib directly.

Reply:
@dheepak-aot can you please collaborate on this?

Reply:
I agree with @andrewsignori-aot, current scope of the services that serve the external APIs can be in `API` scope.